packages/client-config/src/client-config-schema/schema_v1.json

```diff
@@ -0,0 +1,401 @@
+        }
+      },
+      "required": [
+        "aws_region",
+        "url",
+        "default_authorization_type",
+        "authorization_types"
+      ]
+    },
+    "geo": {
+      "description": "Outputs manually specified by developers for use with frontend library",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "aws_region": {
+          "description": "AWS Region of Amazon Location Service resources",
+          "$ref": "#/$defs/aws_region"
+        },
+        "maps": {
+          "description": "Maps from Amazon Location Service",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "items": {
+              "type": "array",
+              "items": {
+                "$ref": "#/$defs/amazon_location_service_config"
+              }
+            },
+            "default": {
+              "type": "string"
+            }
+          },
+          "required": ["items", "default"]
+        },
+        "search_indices": {
+          "description": "Location search (search by places, addresses, coordinates)",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "default": {
+              "type": "string"
+            }
+          },
+          "required": ["items", "default"]
+        },
+        "geofence_collections": {
+          "description": "Geofencing (visualize virtual perimeters)",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "default": {
+              "type": "string"
+            }
+          },
+          "required": ["items", "default"]
+        }
+      },
+      "required": ["aws_region"]
+    },
+    "storage": {
+      "type": "object",
+      "description": "Outputs generated from defineStorage",
+      "additionalProperties": false,
+      "properties": {
+        "aws_region": {
+          "$ref": "#/$defs/aws_region"
+        },
+        "bucket_name": {
+          "type": "string"
+        }
+      },
+      "required": ["aws_region", "bucket_name"]
+    },
+    "custom": {
+      "description": "Outputs generated from backend.addOutput({ custom: <config> })",
+      "type": "object"
+    }
+  },
+  "required": ["version"],
+  "$defs": {
+    "aws_region": {
+      "type": "string"
+    },
+    "amazon_cognito_standard_attributes": {
+      "description": "Amazon Cognito standard attributes for users -- https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html",
+      "type": "string",
+      "enum": [
```

Comment:
how will we keep these enums in sync the the actual available options?

Reply:
I don't think these change very often. if they do, it will be a new version of the config. CC: @josefaidt